package.json

```diff
@@ -0,0 +1,629 @@
+{
```

Comment:
- According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/166856533-1528aa5b-9eda-4cb1-81a5-5d09310ed183.png)

Therefore:

- [ ] Please, ensure that:

- Basic commands for building Webpack in development mode, production mode is configured
- Webpack dev server is set up and running with no errors.

## Explanation

This project should not be run with the "live server" VsCode extension.  This project should be run with the command "npm start".  Also, there should be a way to run the project in "watch" and "production" modes.  You will need to configure webpack and package.json for this.

## Hints

- Add this "scripts" section to your package.json:

![image](https://user-images.githubusercontent.com/57421823/166856833-d1b12788-52da-49c3-a35b-c5d22d710fca.png)

- Install the webpack dev server, CLI, Html plugin, and CSS loader:

```bash
npm install webpack-dev-server --save-dev
npm install -D webpack-cli
npm install --save-dev style-loader
npm install css-loader -D
npm install html-webpack-plugin
```

- Use this webpack config:

```javascript
const path = require('path');
const HtmlWebpackPlugin = require('html-webpack-plugin');

module.exports = {
  mode: 'development',
  entry: './src/index.js',
  devtool: 'inline-source-map',
  devServer: {
    static: './dist',
  },
  plugins: [
    new HtmlWebpackPlugin({
      template: './src/index.html',
    }),
  ],
  output: {
    filename: 'main.js',
    path: path.resolve(__dirname, 'dist'),
  },
  module: {
    rules: [
      {
        test: /\.css$/i,
        use: ['style-loader', 'css-loader'],
      },
    ],
  },
};
```
- [ ] OPTIONAL: I highly suggest getting rid of the hundreds of unnecessary dependencies your project has.